Assets/LeapMotionModules/Interaction/Scripts/UI/InteractionButton.cs

```diff
@@ -72,14 +73,23 @@ public class InteractionButton : InteractionBehaviour {
     private Vector3 _physicsVelocity = Vector3.zero;
     private bool _handIsLeft = false;
     private bool _physicsOccurred;
+    private bool initialIgnoreGrasping = false;
+    private Quaternion initialLocalRotation;
 
     protected override void Start() {
+      if(transform == transform.root) {
+        Debug.LogError("This button has no parent!  Please ensure that it is parented to something!", this);
```

Comment:
I'm gonna be a stickler and say probably only one exclamation mark is necessary here (the second one can be a period and then we're not! yelling as much!)